app/models/publication.rb

```diff
@@ -65,7 +65,13 @@ def self.open_access_statuses
   end
 
   def self.oa_workflow_states
-    ['automatic DOI verification pending']
+    [
+      'automatic DOI verification pending',
+      'oa metadata search pending',
+      'error during oa metadata search',
+      'open access publication',
+      'no open access data found'
```

Comment:
I think we can get rid of `'open access publication'` and `'no open access data found'` now, correct? `'open access publication'` will instead be indicated by the presence of `OpenAccessLocation`s, and `'no open access data found'` isn't a state that we care about any more since we'll be running the check continuously regardless.

I sort of don't think that we need to record `'error during oa metadata search'` either since we're currently not doing anything with that information. If an error does occur and the cause is transient, then active_job should retry the job until it succeeds. And even if none of those retries succeed, it'll be queued up again an hour later. So we'd only need to know about really persistent errors, and I think we can find out about those just by allowing the job to blow up without rescuing anything at all.